examples/src/bin/blinky.rs

```diff
@@ -25,9 +25,7 @@ fn main() -> ! {
 
     // Set GPIO0 as an output, and set its state high initially.
     let io = Io::new(peripherals.GPIO, peripherals.IO_MUX);
-    let mut led = io.pins.gpio8.into_push_pull_output();
-
-    led.set_high();
+    let mut led = Output::new(io.pins.gpio0, true);
```

Comment:
Why gpio0? Description should be updated as well in that case.

Reply:
mhhh the description actually says GPIO0 .... so now it's "right"

Reply:
Oh :D 0 and 8 looks similar in VSCode :D